roles/nv_gpu_install_from_commit/tasks/build.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: find a node with our tuned label
+  command: oc get nodes -l "{{ tuned_module_fuse_label }}" -oname
+  register: fuse_nodes
+- name: find a worker node to label for loading tuned module
+  shell: oc get nodes -l node-role.kubernetes.io/worker -oname | head -1
+  when: fuse_nodes.stdout == ""
+  register: worker_nodes
+- name: mark node with our tuned label
+  command: oc label "{{ worker_nodes.stdout }}" "{{ tuned_module_fuse_label }}="
+  when: fuse_nodes.stdout == ""
+
+- name: apply tuned fuse profile manifest
+  command: oc apply -f "{{ tuned_module_fuse }}"
+
+- name: apply ci utils manifest
+  command: oc apply -f "{{ gpu_operator_ci_utils }}"
+
+- name: authorize ns/gpu-operator-ci Pods to access ns/gpu-operator-ci-utils images
+  command: oc policy add-role-to-user \
+             system:image-puller system:serviceaccount:gpu-operator-ci:default \
+             --namespace=gpu-operator-ci-utils
+
+- name: apply namespace manifest
+  command: oc apply -f "{{ gpu_operator_namespace }}"
+
+- name: apply imagestream manifest
+  command: oc apply -f "{{ gpu_operator_imagestream }}"
+
+- name: apply image helper builder manifest
+  command: oc apply -f "{{ gpu_operator_helper_image_builder }}"
+
+- name: wait for the helper image to be built
+  command: oc logs -f bc/helper-image-builder -n gpu-operator-ci-utils
+  register: wait_helper_image
+  until: wait_helper_image.rc != 1
+  retries: 10
+  delay: 10
+
+- name: apply operator image builder script manifest
+  command: oc apply -f "{{ gpu_operator_image_builder_script }}"
+
+- name: find the name of the builder-dockercfg secret
+  block:
+    - name: find the name of the builder-dockercfg secret
+      shell: oc get secrets -oname -n gpu-operator-ci | cut -d/ -f2 | grep builder-dockercfg
+      register: builder_secret
+  rescue:
+    - name: Failed to find the builder-dockercfg secret
+      fail:
+        msg: "No builder-dockercfg secret in the 'gpu-operator-ci' namespace ..."
+
+- name: delete operator image builder pod, if any
+  shell: oc delete -f "{{ gpu_operator_image_builder_pod }}" || true
+
+- name: delete operator image, if any
+  shell: oc delete imagestreamtag/gpu-operator-ci:gpu-operator -n gpu-operator-ci || true
+
+- name: apply operator image builder pod manifest
+  shell: sed 's|{{ '{{' }} builder_secret {{ '}}' }}|{{ builder_secret.stdout }}|' "{{ gpu_operator_image_builder_pod }}" \
+       | sed 's|{{ '{{' }} gpu_operator_git_repo {{ '}}' }}|{{ gpu_operator_git_repo }}|' \
+       | sed 's|{{ '{{' }} gpu_operator_git_ref {{ '}}' }}|{{ gpu_operator_git_ref }}|' \
+       | sed 's|{{ '{{' }} gpu_operator_image_tag {{ '}}' }}|{{ gpu_operator_image_tag }}|' \
+       | oc apply -f-
+  args:
+    warn: false # don't warn about using sed here
+
+- name: wait for the operator image to be built
+  shell: oc logs -f pod/operator-image-builder-pod -n gpu-operator-ci
```

Comment:
Same comment as above, I really don't like ansible to watch logs as a way to see if a container run to completion